config/locales/ru.yml

```diff
@@ -1096,7 +1097,8 @@ ru:
     reference: "Ссылка"
     refund: "Возврат"
     refund_amount_must_be_greater_than_zero:
-    refund_reasons: "Причины возврата"
+    refund_reasons: "Причины возврата денег"
+    return_reasons: "Причины возврата товара"
```

Comment:
There is no duplicated lines, they are two different lines:
1099 was wrong
1101 is a new one (I have added this one because it was missing translation)